### DISCUSSION TITLE
[Bug] Why can't I find the code for the adaptive throttling algorithm in dubbo's source code, but I see it in dubbo's documentation.

### DISCUSSION THREAD

**Discussion by Iron-Buster** [apache/dubbo#14000] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar issues.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Dubbo Version

Dubbo Java 3.3.0-beta.1, OpenJDK 1.8, Windows 10

### Steps to reproduce this issue

Open the dubbo project with IDEA and search for heuristicSmoothingFlowControl or autoConcurrencyLimiter.

### What you expected to happen

I think there is a problem with the documentation for Dubbo's adaptive throttling algorithm, and if I want to know more about this algorithm, I think I should be able to search for it in Dubbo's source code. [adaptive-concurrency-control](https://cn.dubbo.apache.org/zh-cn/overview/mannual/java-sdk/advanced-features-and-usage/performance/adaptive-concurrency-control/)

### Anything else

Nothing.

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Iron-Buster** [comment]
Can someone explain why?🥲

**Comment by xxxxxxxxxxiong** [comment]
I also want to know about this, it seems merge request does not be merged  

https://github.com/apache/dubbo/pull/10642

**Reply by chickenlj** [reply]
Yes, @xiongshengjie is right with the current status of this feature. The release of this feature has been delayed for some reason. It's expected to be released in the upcoming 3.3.x version. We will also update the documentation soon.

**Reply by xxxxxxxxxxiong** [reply]
ok, thanks for your reply.
I will follow this feature, and will be glad to see the feature released

**Reply by Iron-Buster** [reply]
thanks for your reply. I'm looking forward to this feature.
